File: DuDoong-Common/src/test/java/band/gosrock/common/properties/JwtPropertiesTest.java
package band.gosrock.common.properties;

import static org.junit.jupiter.api.Assertions.*;

import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;

@SpringBootTest
class JwtPropertiesTest {

    @Autowired
    JwtProperties jwtProperties;

    @Test
    void JWT_환경변수값이_불러와지는지_확인() {
        Long accessExp = jwtProperties.getAccessExp();
        Long refreshExp = jwtProperties.getRefreshExp();
        assertEquals(accessExp, 3600);
        assertEquals(refreshExp, 3600);
    }
}

File: DuDoong-Common/src/test/kotlin/band/gosrock/common/properties/JwtPropertiesTest.kt
package band.gosrock.common.properties

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.test.context.ActiveProfiles

@SpringBootTest
@ActiveProfiles("common")
class JwtPropertiesTest {

    @Autowired
    lateinit var jwtProperties: JwtProperties

    @Test
    fun `JWT 환경변수값이 불러와지는지 확인`() {
        val accessExp = jwtProperties.accessExp
        val refreshExp = jwtProperties.refreshExp
        assertEquals(accessExp, 3600)
        assertEquals(refreshExp, 3600)
    }
}
